apps/freeablo/faworld/playerstats.cpp

```diff
@@ -1,6 +1,7 @@
 #include "playerstats.h"
 #include "../fasavegame/gameloader.h"
 #include "diabloexe/characterstats.h"
+#include <algorithm>
```

Comment:
Was this intentional?

Reply:
Yeah fixed windows build (`error C2039: 'min': is not a member of 'std'`), probably to do with the `NOMINMAX` flag